ephios/static/ephios/js/group_form.js

```diff
@@ -1,11 +1,31 @@
 $(document).ready(function () {
-    $("#id_can_add_event").click(function (event) {
+    $("#id_is_planning_group").click(function (event) {
         if ($(this).is(":checked"))
             $(".publish-select").slideDown();
         else
             $(".publish-select").slideUp();
     });
-    if (!$("#id_can_add_event").is(":checked")) {
+    if (!$("#id_is_planning_group").is(":checked")) {
         $(".publish-select").slideUp();
     }
+    $("#id_is_management_group").click(function (event) {
+        if ($(this).is(":checked")) {
+            $("#id_is_hr_group").prop("disabled", true);
+            $("#id_is_hr_group").prop("checked", true);
+            $("#id_is_planning_group").prop("disabled", true);
+            $("#id_is_planning_group").prop("checked", true);
+            $(".publish-select").slideDown();
+        }
+        else {
+            $("#id_is_hr_group").prop("disabled", false);
+            $("#id_is_hr_group").prop("checked", false);
+            $("#id_is_planning_group").prop("disabled", false);
+            $("#id_is_planning_group").prop("checked", false);
```

Comment:
I think we should leave the two checkboxes checked.

Reply:
or set them to their initial value when available

Reply:
hm i think i like it this way. when editing a group with management permissions and revoking the management permission it feels more natural to revoke the included permissions as well

Reply:
I didn't think of the other being "included" but being obligatory when being a manager, and I still believe that's the more sensible way of thinking about it when showing them all as seperate checkboxes.

Reply:
but I never actually checked them, so why do I need to uncheck them once I want to revoke the management permission?

Reply:
There probably is no right answer here. Well maybe you did check them, maybe you didn't. I'd say the more common use case is to only revoke some permissions, not all of them at once.